config/authors/ndubuisiaso.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Aso Orji",
+  "bio": "Software Engineer || Flutter Developer",
```

Comment:
Aso is a software engineer with three years of experience designing and developing databases and mobile applications, He's an engineer skilled in all phases of software development and algorithm implementation.